sonar-plugin-api/src/test/java/org/sonar/api/server/rule/RulesDefinitionXmlLoaderTest.java

```diff
@@ -57,6 +57,9 @@ public void parse_xml() throws Exception {
     assertThat(rule.severity()).isEqualTo(Severity.BLOCKER);
     assertThat(rule.template()).isTrue();
     assertThat(rule.status()).isEqualTo(RuleStatus.BETA);
+    assertThat(rule.debtSubCharacteristic()).isEqualTo("INTEGRATION_TESTABILITY");
+    assertThat(rule.debtRemediationFunction().coefficient()).isEqualTo("2d 2h 2min".replaceAll(" ", ""));
```

Comment:
Please remove this useless .replaceAll(" ", ""), and replace expected result by "2d2h2min"